client/src/actions/userActions.js

```diff
@@ -101,8 +101,8 @@ export const register = async (dispatch, registerPayload) => {
 // User logout action
 export const logout = async (dispatch) => {
   localStorage.removeItem("userInfo");
-
   dispatch({
     type: USER_LOGOUT,
   });
+  await axios.get("/api/users/logout");

```

Comment:
You'll probably want to not remove the userInfo from localstorage if this request fails